app/src/main/java/com/mapbox/maps/testapp/examples/RestrictBoundsActivity.kt

```diff
@@ -130,37 +132,48 @@ class RestrictBoundsActivity : AppCompatActivity() {
 
   companion object {
     private val BOUNDS_ID = "BOUNDS_ID"
-    private val ICELAND_BOUNDS: BoundOptions = BoundOptions.Builder()
+    private val ICELAND_BOUNDS: CameraBoundsOptions = CameraBoundsOptions.Builder()
       .bounds(
         CoordinateBounds(
           Point.fromLngLat(-25.985652, 62.985661),
-          Point.fromLngLat(-12.626277, 66.852863)
+          Point.fromLngLat(-12.626277, 66.852863),
+          false
         )
       )
       .minZoom(2.0)
       .build()
 
-    private val ALMOST_WORLD_BOUNDS: BoundOptions = BoundOptions.Builder()
+    private val ALMOST_WORLD_BOUNDS: CameraBoundsOptions = CameraBoundsOptions.Builder()
       .bounds(
         CoordinateBounds(
           Point.fromLngLat(-170.0, -20.0),
-          Point.fromLngLat(170.0, 20.0)
+          Point.fromLngLat(170.0, 20.0),
+          false
         )
       )
       .minZoom(2.0)
       .build()
 
     @SuppressLint("Range")
-    private val CROSS_IDL_BOUNDS: BoundOptions = BoundOptions.Builder()
+    private val CROSS_IDL_BOUNDS: CameraBoundsOptions = CameraBoundsOptions.Builder()
       .bounds(
         CoordinateBounds(
           Point.fromLngLat(170.0202020, -20.0),
-          Point.fromLngLat(190.0, 20.0)
+          Point.fromLngLat(190.0, 20.0),
+          false
         )
       )
       .minZoom(2.0)
       .build()
 
-    private val INFINITE_BOUNDS: BoundOptions = BoundOptions.Builder().infiniteBounds(true).build()
+    private val INFINITE_BOUNDS: CameraBoundsOptions = CameraBoundsOptions.Builder()
+      .bounds(
+        CoordinateBounds(
+          Point.fromLngLat(170.0202020, -20.0),
+          Point.fromLngLat(190.0, 20.0),
```

Comment:
why these two points?

Reply:
as per docs if infinite bounds flag is `true` those values are not taken into account. so just took any values 😄